go/integTest/shared_commands_test.go

```diff
@@ -1040,6 +1040,39 @@ func (suite *GlideTestSuite) TestHStrLen_WithNotExistingField() {
 	})
 }
 
+func (suite *GlideTestSuite) TestHIncrByFloat_WithExistingField() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := uuid.NewString()
+		field := uuid.NewString()
+		fieldValueMap := map[string]string{field: "10"}
+
+		hsetResult, err := client.HSet(key, fieldValueMap)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(1), hsetResult.Value())
+
+		hincrByFloatResult, hincrByFloatErr := client.HIncrByFloat(key, field, 1.5)
+		assert.Nil(suite.T(), hincrByFloatErr)
+		assert.Equal(suite.T(), float64(11.5), hincrByFloatResult.Value())
+	})
+}
+
+func (suite *GlideTestSuite) TestHIncrByFloat_WithNonExistingField() {
```

Comment:
do these need to be 2 different tests?

Reply:
We can merge them but that does not decrease the number of lines. Plus, I am following the pattern for rest of tests in this file.

Also, I think it is always good to avoid creating large tests. Small tests are easy to understand.